config/config.yaml.dist

```diff
@@ -479,3 +479,6 @@ cicd:
 #secret:
 #    tls:
 #        defaultValidity: 8760h # 1 year
+
+enterprise:
+    enabled: false
```

Comment:
Default config, so it should be commented. Also, default value has to be added to Viper.

```suggestion
#enterprise:
#    enabled: false
```

Furthermore, I think something like this would be better:

```yaml
pipeline:
    enterprise: false
````